src/device-registry/bin/www

```diff
@@ -49,6 +105,14 @@ cluster(function() {
     //Create HTTP server.
     //Listen on provided port, on all network interfaces.
     server.listen(port, function() {
+        /**
+         * only run KAFKA in staging for now --- as we continue to do QA tests
```

Comment:
To continuously consume KAFKA messages, calling the run() function on server start-up, in the **server.listen()** callback function. @BenjaminSsempala